package.json

```diff
@@ -34,9 +34,6 @@
     "activationEvents": [
         "*"
     ],
-    "extensionDependencies": [
```

Comment:
@arjun27: I removed the Live Share extension dependency as I think it is no longer required. Maybe that is isn't correct though?